src/service.js

```diff
@@ -34,7 +44,7 @@
 
         // Check if grecaptcha is not defined already.
         if (ng.isDefined($window.grecaptcha)) {
-            $window.vcRecaptchaApiLoaded();
+            $window.vcRecaptchaApiLoadedCallback();
```

Comment:
This would end up getting called by each of the instances, which in turn would trigger the resolve on 16 getting called multiple times.